texts/navigation.yml

```diff
@@ -4,7 +4,7 @@ items:
   index: Overview
   data: Data
   content: Content
-  deposit-dates: Deposit compliance
+  compliance: Compliance
```

Comment:
Please, keep this as it was because it was not discussed with the team.

I used shorter word on the mockup because I would need to change the sidebar size on every mockup in the project otherwise.

Reply:
I thought you discussed this with David since it was in design

Will revert it.